manifests/macvtap.yaml

```diff
@@ -33,6 +33,22 @@ spec:
           - name: deviceplugin
             mountPath: /var/lib/kubelet/device-plugins
         terminationMessagePolicy: FallbackToLogsOnError
+        readinessProbe:
+          exec:
+            command:
+            - sh
+            - -c
+            - test -S /var/lib/kubelet/device-plugins/macvtap.network.kubevirt.io*
+          initialDelaySeconds: 5
+          periodSeconds: 10
+        livenessProbe:
+          exec:
+            command:
+            - sh
+            - -c
+            - test -S /var/lib/kubelet/device-plugins/macvtap.network.kubevirt.io*
+          initialDelaySeconds: 15
+          periodSeconds: 20
```

Comment:
maybe make this less frequent ? once per minute maybe ?

Reply:
sure